src/java/picard/analysis/directed/CollectHsMetrics.java

```diff
@@ -45,15 +45,76 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Collects a set of Hybrid Selection specific metrics from an aligned SAM" +
-                "or BAM file. If a reference sequence is provided, AT/GC dropout metrics will " +
-                "be calculated, and the PER_TARGET_COVERAGE option can be used to output GC and " +
-                "mean coverage information for every target.",
-        usageShort = "Collects Hybrid Selection-specific metrics for a SAM or BAM file",
+        usage = CollectHsMetrics.USAGE_SUMMARY + CollectHsMetrics.USAGE_DETAILS,
+        usageShort = CollectHsMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
+
 )
 public class CollectHsMetrics extends CollectTargetedMetrics<HsMetrics, HsMetricCollector> {
 
+    static final String USAGE_SUMMARY = "Collects hybrid-selection (HS) specific metrics for a SAM or BAM file.  ";
+    static final String USAGE_DETAILS = "These metrics enable users to determine the efficacy of HS experiments.  " +
+            "In addition, if a reference sequence is provided, this program will calculate AT/GC dropout metrics and enable users to invoke the" +
+            " PER_TARGET_COVERAGE option to output both GC and mean coverage information for every target. <br /><br />" +
+            "" +
+            "Both CollectTargetedPCRMetrics and CalculateHybridSelection metrics share virtually identical program structures except " +
+            "for the name of their targeting mechanisms (e.g. bait set or amplicon set).  Both depend on the TargetMetricsCollector " +
+            "and collect metrics on all reads in the INPUT SAM/BAM file. <br /><br /> " +
+            "" +
+            "Hybrid selection (HS) enables targeted sequencing analysis via the capture of specified genomic DNA " +
+            "sequences (doi:10.1038/nbt.1523).  It is commonly used to characterize exon sequences from genomic DNA " +
+            "or filter out bacterial DNA sequences from clinical samples.  <br /><br /> " +
+            "" +
+            "The technique involves the capture of unique regions of genomic DNA (targets) using synthetic RNA baits." +
+            "   The baits are synthesized with biotinylated nucleotides to facilitate capture of bait:target hybrids on" +
+            " streptavidin beads.  The captured target sequences are amplified, sequenced, and processed for variant calling. <br /><br />" +
+            "" +
+            "This tool requires an aligned SAM or BAM file as well as bait and target interval files.  " +
+            "The bait and target files can be formatted as VCF or as interval_list files.  For information on VCF files, please see:" +
```

Comment:
the format of a VCF doesn't informa  us to use a vcf for specifying intervals. perhaps a small example would help.

Also, since you are mentioning intervalList, please say a few words on the format or point to a place (intervalListTools?) where the format is specified.